c2rust-transpile/src/diagnostics.rs

```diff
@@ -64,7 +64,9 @@ pub fn init(mut enabled_warnings: HashSet<Diagnostic>, log_level: log::LevelFilt
         })
         .chain(io::stderr())
         .into_log();
+    let _ = log_reroute::init(); // Ignore the [`SetLoggerError`] b/c we just want to make sure it's set at least once.
```

Comment:
I think you should make this explicitly `let _ignored: Result<(), SetLoggerError> = ...` so that if log_reroute ever returns a new error, we think about whether we want to check for it.

Reply:
```suggestion
    // Ignore the [`SetLoggerError`] b/c we just want to make sure it's set at least once.
    let _: Result<(), SetLoggerError> = log_reroute::init();
```

How about this?  `_` vs. `_ignored` have different drop semantics, but putting the type annotation there ensures it won't change without a compile error.

Reply:
Yep, that looks good. I didn't actually know you could attach a type annotation to `_`. Makes sense, but TIL.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/615/commits/d7508f925555d72eccba536307b0943769c0fe15.